linux_os/guide/system/software/integrity/crypto/harden_sshd_macs_opensshserver_conf_crypto_policy/tests/rhel9_stig_incorrect_policy.fail.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# platform = Oracle Linux 9,Red Hat Enterprise Linux 9
+# variables = sshd_approved_macs=hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512,hmac-sha2-256
+# remediation = none
+
+configfile=/etc/crypto-policies/back-ends/opensshserver.config
+
+# Ensure directory + file is there
+test -d /etc/crypto-policies/back-ends || mkdir -p /etc/crypto-policies/back-ends
+
+if [[ -f $configfile ]]; then
+    sed -i -r "s/^(?!\s*#)MACs\s+\S+/MACs hmac-sha2-512,hmac-sha2-256,hmac-sha1,hmac-sha1-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com/" $configfile
```

Comment:
```suggestion
    sed -i -r "s/MACs\s+\S+/MACs hmac-sha2-512,hmac-sha2-256,hmac-sha1,hmac-sha1-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com/" $configfile
```

Similarly as in: https://github.com/ComplianceAsCode/content/blob/92283ec70396250f1d1257a18c77f1c2c7503180/linux_os/guide/system/software/integrity/crypto/harden_sshd_ciphers_opensshserver_conf_crypto_policy/tests/rhel9_stig_incorrect_policy.fail.sh#L12

The test is failing to setup on my testing and in the gating

Reply:
Fixed